roles/pip/tasks/main.yml

```diff
@@ -32,3 +28,13 @@
       - { name: pip, version: "{{ pip_version }}"}
       - { name: requests, version: "{{ requests_version }}"}
       - { name: pyOpenSSL, version: "{{ pyopenssl_version }}"}
+
+  - name: upgrade pip to fix broken pip2 on debian flex
+    shell:
+      cmd: python -m pip install --upgrade pip
+    when: ansible_distribution_release == 'Flex'
```

Comment:
Just a small thing, I'd add `ansible_distribution == Debian` here as well to make it more clear.